cmd/build.go

```diff
@@ -47,11 +46,13 @@ var buildCmd = &cobra.Command{
 func init() {
 	Promu.AddCommand(buildCmd)
 
+	buildCmd.Flags().Bool("cgo", false, "Enable CGO")
```

Comment:
It might be helpful to give some more details here. If I understand the change correctly, this will lead to  different build images being used, right?